stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/plugin/OfflinePlugin.kt

```diff
@@ -78,5 +78,6 @@ public class OfflinePlugin(private val config: Config) : Plugin {
 
     public companion object {
         public const val MODULE_NAME: String = "Offline"
+        public const val TOGGLE_KEY_OFFLINE: String = "offline plugin toggle"
```

Comment:
NIT:
```suggestion
        public const val TOGGLE_KEY_OFFLINE: String = "Offline Plugin"
```

Reply:
Also, I would move it to the `ToggleService` as constant will be used only for toggling